Silicon/Ampere/AmpereAltraPkg/AmpereAltraPkg.dsc.inc

```diff
@@ -597,7 +597,6 @@
       NULL|MdeModulePkg/Library/LzmaCustomDecompressLib/LzmaCustomDecompressLib.inf
   }
   MdeModulePkg/Universal/ReportStatusCodeRouter/Pei/ReportStatusCodeRouterPei.inf
-  MdeModulePkg/Universal/StatusCodeHandler/Pei/StatusCodeHandlerPei.inf

```

Comment:
I think the definition in the parent DSC i.e ComHpcAlt.dsc should override it

Reply:
I've updated Jade.dsc to fix the build.